src/lib/cache.svelte.ts

```
@@ -0,0 +1,187 @@
+
+// Households
+const {data: householdsData, error: householdsError} = await supabase
+	.from("households")
+	.select("id, name, head")
+	.order("name")
+if (householdsError) { console.error("Error in getting households:", householdsError); toast.error("Error in getting households.") }
+else householdsData.forEach(household => households[household.id] = {
+	...household,
+	people: {},
+	meals: {},
+})
```

Biome (1.9.4)

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)